data/pirate jobs.txt

```diff
@@ -0,0 +1,2750 @@
+		government Republic
+		fleet "Small Republic"
+	npc accompany save
+		government Pirate
+		personality escort
+		fleet
+			names "pirate"
+			variant 2
+				"Headhunter"
+			variant 2
+				"Headhunter (Particle)"
+			variant 2
+				"Raven (Afterburner)"
+			variant 2
+				"Raven (Heavy)"
+			variant 1
+				"Corvette"
+			variant 1
+				"Corvette (Missile)"
+			variant 1
+				"Corvette (Speedy)"
+	on complete
+		payment 180000
+		dialog "The captain of the <npc> thanks you for escorting them safely, and pays you <payment>."
+	on visit
+		dialog "You have reached <planet>, but you left the <npc> behind! Better depart and wait for them to arrive in this star system."
+
+mission "Escort Illegal Cargo (Large, North, Dangerous Path)"
+	name "Protect cargo to <planet>"
+	job
+	repeat
+	description "The captain of the <npc> will pay you <payment> to escort them and their illegal cargo safely through law enforced space to reach <destination> by <date>."
+	deadline 12
+	to offer
+		random < 30
+		"combat rating" > 600
+		"reputation: Pirate" > 0
+	source
+		government "Pirate"
+		attributes "north pirate"
+	destination
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes frontier north
+		distance 3 6
+	npc
+		system destination
+		personality staying
+		government Republic
+		fleet "Large Republic"
+	npc accompany save
+		government Pirate
+		personality escort
+		fleet
+			names "pirate"
+			variant 3
+				"Corvette"
+			variant 3
+				"Corvette (Missile)"
+			variant 3
+				"Corvette (Speedy)"
+			variant 2
+				"Firebird"
+			variant 2
+				"Firebird (Missile)"
+			variant 2
+				"Firebird (Plasma)"
+			variant 1
+				"Leviathan"
+	on complete
+		payment 240000
+		dialog "The captain of the <npc> thanks you for escorting them safely, and pays you <payment>."
+	on visit
+		dialog "You have reached <planet>, but you left the <npc> behind! Better depart and wait for them to arrive in this star system."
+
+mission "Escort Stolen Vessel (Extra Large, North, Dangerous Path)"
+	name "Protect ship to <planet>"
+	job
+	repeat
+	description "The captain of the <npc> will pay you <payment> to escort them and their stolen vessel safely through law enforced space to reach <destination> by <date>."
+	deadline 12
+	to offer
+		random < 15
+		"combat rating" > 1000
+		"reputation: Pirate" > 0
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes frontier north
+	destination
+		government "Pirate"
+		attributes "north pirate"
+		distance 3 6
+	npc
+		system destination
+		personality staying
+		government Republic
+		fleet "Small Republic"
+	npc
+		system destination
+		personality staying
+		government Republic
+		fleet "Large Republic"
+	npc accompany save
+		government Pirate
+		personality escort
+		fleet
+			names "pirate"
+			variant 3
+				"Leviathan"
+				"Hauler II"
+			variant 2
+				"Leviathan (Laser)"
+				"Hauler"
+			variant 2
+				"Leviathan (Heavy)"
+				"Freighter"
+			variant 1
+				"Bactrian"
+				"Dagger" 3
```

Comment:
You might not want to include fighters here, because if any one of them dies or gets left behind the mission will fail.